src/specs/01_basic/raft_gas_usage.spec

```diff
@@ -31,11 +31,11 @@ Note that intrinsic gas for the simple contract is approx 57352 and total requir
 Privacy enhancements are not currently deducting the intrinsic gas so these transactions fail after the transaction is
 successfully simulated and minted (the behavior is the same as if privacy enhancements are disabled).
 * Private transaction where minter is a participant and gas value is "60352", name this contract as "contract3"
-* Contract "contract3" had exception with message "not-enough gas"
+* Contract "contract3" had exception with message "Gas used: 0. Revert reason: 'N/A'."
 * No transactions are pending on node for "contract3"
 
 * Private transaction where minter is not a participant and gas value is "60352", name this contract as "contract4"
-* Contract "contract4" had exception with message "not-enough gas"
+* Contract "contract4" had exception with message "Gas used: 0. Revert reason: 'N/A'."
```

Comment:
why the message changed? as the error not enough gas is specific on the reason, while now it has N/A

Reply:
We discussed it with @nmvalera and the new wording is like that, and that regex match is as good as it gets when the contract fails.

Reply:
I'm only checking locally why is returning that